### DISCUSSION TITLE
[Q&A] NVFlare/tree/main/research/quantifying-data-leakage  cifar10

### DISCUSSION THREAD

**Discussion by mstc-xqp** [NVIDIA/NVFlare#3112] [Q&A]
### Python version (`python3 -V`)

3.10

### NVFlare version (`python3 -m pip list | grep "nvflare"`)

2.5

### NVFlare branch (if running examples, please use the branch that corresponds to the NVFlare version, `git branch`)

2.5

### Operating system

22.04

### Have you successfully run any of the following examples?

- [ ] hello-numpy-sag with simulator
- [ ] hello-pt with simulator
- [ ] hello-numpy-sag with POC
- [ ] hello-pt with POC

### Please describe your question

I run into the following issue and don't know what to do ...
"I'd like to modify this project to work with the CIFAR-10 dataset instead of the chest X-ray dataset. Could you help me adapt the code for CIFAR-10?"

**Comment by chesterxgchen** [comment]
@holgerroth

**Comment by holgerroth** [comment]
Hi, sorry for the late reply. You should be able to run the code using the CIFAR10Learner instead of [CXRLearner](https://github.com/NVIDIA/NVFlare/blob/fe41433a2ca6f2047fad9745d741ebb365e3940c/research/quantifying-data-leakage/jobs/app_template/client/config/config_fed_client_inv.json#L54). You can take the cifar 10 code from [here](https://github.com/NVIDIA/NVFlare/blob/main/examples/advanced/cifar10/pt/learners/cifar10_learner.py). Just make sure that the inversion code uses the same model architecture as your learner code.
